tools/ci_build/build.py

```diff
@@ -761,15 +764,17 @@ def generate_build_tree(cmake_path, source_dir, build_dir, cuda_home, cudnn_home
                 args.android or (args.ios and is_macOS())
                 or args.use_rknpu)
             else "OFF"),
-        "-Donnxruntime_USE_TVM=" + ("ON" if (args.use_nuphar or args.use_stvm) else "OFF"),
-        "-Donnxruntime_USE_LLVM=" + ("ON" if (args.use_nuphar or args.use_stvm) else "OFF"),
+        "-Donnxruntime_USE_NUPHAR_TVM=" + ("ON" if args.use_nuphar else "OFF"),
+        # TODO(vvchernov): do we constantly use LLVM if we use TVM EP? if we need CUDA for TVM EP?
```

Comment:
The default configuration can be different of CPU and CUDA. The most efficient should be chosen.

Reply:
It was TODO for check of possible building pipeline. I discussed it with my colleagues and LLVM is needed in all cases. TODO was removed.